### ISSUE TITLE
Use some kind of build system to run unit tests

### ISSUE THREAD

**Issue by jakubtuchol** [derekargueta/Interview-Prep#1]
While it's probably not the core point of this repo, you really should use some kind of build system, such as [Maven](https://maven.apache.org/) or [Gradle](http://gradle.org/) to automate testing. This will save you much pain and time.

**Comment by derekargueta** [comment]
Agreed. I'm not a big Java dev besides AP CS (iOS by day and Python by night) but I'm trying to get more familiar with the "Java way". I'll look into integrating Gradle. Thanks!

**Comment by jakubtuchol** [comment]
I'm working on a refactor to get this working with Maven. Haven't really looked into Gradle, but I'll try to have the PR out in the next day or so.
